### PR TITLE
Added social Links  in  footer

### LINKED ISSUE #94: Implement Social Footer Section
image

To use the already included icon set, the icons should already be inside.

/triage accepted
/area ui-ux layouts

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Social media links on footer
#### Which issue(s) does this PR fixes?:

Fixes #94 

### Screenshot

image

image

### CODE REVIEW

File: src/layouts/components/Footer.tsx [hunk 1]
```diff
@@ -25,8 +34,45 @@ export function Footer (): JSX.Element {
               <FooterExternalLink url='https://dex.defichain.com/mainnet/anchors' text='BTC Anchors' />
             </div>
           </div>
+          <div className='flex-1'>
+            <div className='text-2xl font-semibold'>Social</div>
+            <div className='flex flex-wrap mt-3  w-84'>
+              <div className='flex flex-row space-x-2 items-center w-1/2'>
+                <FaTwitterSquare size={25} />
```
Comment (fuxingloh) [inline src/layouts/components/Footer.tsx:41]:
```suggestion
                <FaTwitterSquare size={24} />
```

Always try to use even numbers due to down/up scale algorithm where odd multiple won't look as favorable due to 2x, 3x screens

File: src/layouts/components/Footer.tsx [hunk 2]
```diff
@@ -25,8 +34,45 @@ export function Footer (): JSX.Element {
               <FooterExternalLink url='https://dex.defichain.com/mainnet/anchors' text='BTC Anchors' />
             </div>
           </div>
+          <div className='flex-1'>
+            <div className='text-2xl font-semibold'>Social</div>
+            <div className='flex flex-wrap mt-3  w-84'>
```
Comment (fuxingloh) [inline src/layouts/components/Footer.tsx:39]: Extract this part out into a JSX function `FooterSocialLink` and put it below.

**Review by fuxingloh — CHANGES_REQUESTED** [review]

**Comment by siradji** [comment]
@aikchun

**Comment by siradji** [comment]
@fuxing all changes are done.

**Review by fuxingloh — APPROVED** [review]
- I refactored the remaining sections into function as well. 
- Also refactored my Sitemap to follow your sizing on socials. 
- Additionally, I dropped the use of `md:flex-row flex-col` breakpoint should control overflow not orientetation. 
- I also updated some styles due to recent design changes from Jon

**Pull request merged**